policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/age_threshold.yaml

```diff
@@ -0,0 +1,32 @@
+description: California Foster Youth Tax Credit is limited to person between 18 to 25 and has a maximum amount of 1083
+brackets:  
+  - threshold:  
+      2021-01-01: 0  
+    amount:  
+      2021-01-01: 0  
+  - threshold:  
+      2021-01-01: 18  
+    amount:  
+      #2021-01-01: 25_000
+      2021-01-01: 1_000
```

Comment:
make this the value of the legal code (1,176) - in the variable multiply by policyengine_us/parameters/gov/states/ca/tax/income/credits/earned_income/adjustment/factor.yaml